Hush/Assets/Scripts/Keys/KeyCollect.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class KeyCollect : MonoBehaviour
+{
+
+    private static int numOfKeys = 0;
+    public static int NumOfKeys {
+        get {
+            return numOfKeys;
+        }
+    }
+    public static int keys = 0;
+
+    void OnTriggerEnter(Collider collider) {
+        if (collider.gameObject.name == "Player") {
```

Comment:
Probably wanna compare tags instead of gameObject names... This will break super easily! We have a player tag you can use! Also do we want the player to pickup the key automagically or only when they press the interact button ?